odc/index/__init__.py

```diff
@@ -14,14 +16,15 @@
     month_range,
     ordered_dss,
     parse_doc_stream,
-    patch_urls,
     product_from_yaml,
     season_range,
     time_range,
 )
 from ._utm import mk_utm_gs, utm_region_code, utm_tile_dss, utm_zone_to_epsg
 from ._uuid import odc_uuid
 
+warnings.warn("odc.index is being removed", FutureWarning)
```

Comment:
Could/should this be a https://docs.python.org/3/library/exceptions.html#DeprecationWarning?

Reply:
I wanted it to be louder than `DeprecationWarning`, basically `DeprecationWarning` is hidden by default and I wanted it to show up in your typical notebook workflow in case those things are used in any sample notebooks